exchange/src/randomness/brownian.hpp

```diff
@@ -0,0 +1,69 @@
+/**
+ * @file brownian.hpp
+ * @author Andrew Li (andrewli@u.northwestern.edu)
+ * @brief Brownian motion randomness to simulate market chaos
+ * @version 0.1
+ * @date 2024-01-12
+ *
+ * @copyright Copyright (c) 2024
+ *
+ */
+
+#pragma once
+
+#include <random>
+#include <vector>
+
+namespace nutc {
+
+namespace brownian {
+
+const double BROWNIAN_MOTION_DEVIATION = 0.4;
+
+class BrownianMotion {
+    unsigned int seed;
+    double cur_value;
+
+    std::mt19937 random_number_generator;
+
+public:
+    // dConstructor for BrownianMotion, takes nothing
+    BrownianMotion()
+    {
+        std::random_device rd;
+        random_number_generator = std::mt19937(rd());
+    }
+
+    // Constructor for BrownianMotion, takes a seed
+    BrownianMotion(const unsigned int seed) : seed(seed)
+    {
+        random_number_generator = std::mt19937(seed);
+    }
```

Comment:
This doesn't initialize the `cur_value` field

Also these should be marked `explicit`